docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+Rebuilding an *Admin Workstation* USB
+-------------------------------------
+
+In cases where an *Admin Workstation* USB stick has been lost or destroyed, and no
+backup exists, it is possible to rebuild one. In order to do so, you'll need
+ 
+ - physical access to the SecureDrop servers
+ - 2 USB sticks:
+
+   - Tails Master USB
+   - 1 replacement *Admin Workstation* USB (USB3 and 16GB or better recommended)
+
+The process requires experience with the Linux command line and Tails, and 
+can take up to 3 hours. If a backup of the SecureDrop application server is available,
+reinstalling the instance may be simpler. An outline of the steps involved in
+rebuilding an *Admin Workstation* is as follows:
+
+ #. Prepare the USB sticks.
+ #. (Optional) Boot the *Application* and *Monitor Server* in single user mode and  reset
+    the shell admin account password.
+ #. Set up SSH access for the new *Admin Workstation*.
+ #. Retrieve SecureDrop configuration settings from the  *Application* and *Monitor Server*.
+ #. Back up and configure the SecureDrop application.
+ #. Run the ``./securedrop-admin install`` and ``./securedrop-admintailsconfig``
+    commands from the new *Admin Workstation*.
+ #. Complete post-rebuild tasks.
+
+
+Step 1: Prepare the USB sticks
+==============================
+
+First, create a Tails Master USB using the installation procedure described here: 
+https://tails.boum.org/install/index.en.html
+
+Next, using the Tails Master USB, :ref:`create a new Admin Workstation USB <setup_install_tails>` 
+and set up a persistent volume with a strong passphrase.
+
+Once persistence has been set up, start up the *Admin Workstation* with 
+persistence enabled, :ref:`install the SecureDrop application code, and set up
+the KeePassX database <set_up_admin_tails>`.
+
+The *Admin Workstation* uses SSH with key authentication to connect to the servers,
+so you'll need to create a new SSH keypair for your SecureDrop instance. To do so,
+open a Terminal and run the following command:
+
+.. code:: sh
+
+ ssh-keygen -t rsa -b 4096
```

Comment:
It also prompts for a passphrase, so we should repeat or include our recommendation from https://docs.securedrop.org/en/release-0.13.0/servers.html#set-up-ssh-keys

Reply:
fixed.